.flowconfig

```diff
@@ -11,8 +11,6 @@
 include_warnings=true
 suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe
 suppress_comment=\\(.\\|\n\\)*\\$FlowIssue
-# necessary for webpack loader syntax.
-module.name_mapper='^.*\.css$' -> '<PROJECT_ROOT>/flow/stubs/CSSModuleStub.js'
```

Comment:
We no longer need this custom loader since webpack is now abstracted from us via Create React App. It also failed to work with `css-modules-flow-types-cli`